### PR TITLE
Update item quantity on cart

### PR DESCRIPTION
**Pull request by lehone-hp**

### CODE REVIEW

**Comment by lehone-hp** [comment]
@muarachmann  review

**Comment by lehone-hp** [comment]
Which slider is that boy?

File: public/js/main.js [hunk 1]
```diff
@@ -1,217 +1,227 @@
...
+    var $window = $(window);
+    $window.on('scroll', function() {
+        var scroll = $window.scrollTop();
+        if (scroll < 300) {
+            $(".sticker").removeClass("stick");
+        }else{
+            $(".sticker").addClass("stick");
+        }
+    });
+    /*-- Mobile Menu --*/
```
Comment (muarachmann) [inline public/js/main.js]: meaning here

**Comment by muarachmann** [comment]
Also can i integrate coupon functionality? if yes i could rapidly make a PR for that. So we get to payments. But i think we should host the site now

File: public/js/main.js (hunk 1 above)
Comment (lehone-hp) [reply public/js/main.js]: Ahhh! no. That statement was already there from the template.
I did ctrl+A -> ctrl-X -> ctrl-V on the main.js file so it aligned the whole file, adding spaces before some statements that's why. Na phpstorm nor boy.

**Comment by lehone-hp** [comment]
I have removed the coupon form and the update button

**Comment by muarachmann** [comment]
ok then

**Comment by lehone-hp** [comment]
Now it is to integrate the pay pal right

**Comment by muarachmann** [comment]
pr ok dont forget to place contact information like Phone, address etc and handle contact form too. We left with payment and just inserting of content then i guess

**Comment by lehone-hp** [comment]
yeah

**Comment by muarachmann** [comment]
> Now it is to integrate the pay pal right

yep its simple

**Comment by lehone-hp** [comment]
> > Now it is to integrate the pay pal right
> 
> yep its simple

alright, so how do we proceed

**Comment by muarachmann** [comment]
whatsapp boy! ma data go bole

**Comment by lehone-hp** [comment]
okay okay

**Pull request merged**